src/ckeditor/ckeditor.component.spec.ts

```diff
@@ -423,7 +416,7 @@ describe( 'CKEditorComponent', () => {
 						return eventPromise;
 					} );
 
-					it( 'drag/drop events should emit component dragStart, dragEnd and drop', async done => {
+					it( 'drag/drop events should emit component dragStart, dragEnd and drop', done => {
```

Comment:
Correct me if I'm wrong, but I suppose this test could also be rewritten to return a promise instead of using `done()` method? Then all tests would be nicely unified.

Reply:
It seems like this function shouldn't be async at all. The whole code is synchronous and we are not waiting for editor data.